bin/undelete.py

```diff
@@ -48,8 +48,8 @@ def main():
             if f['name'] == args.filename:
                 del f['deleted']
                 break
-            else:
-                raise RuntimeError('Cannot find {}/{}/{}'.format(args.cont_name, args.cont_id, args.filename))
+        else:
```

Comment:
Nice, Ryan! I've never used a for/else before, I'm used to using a "found" bool. 

http://book.pythontips.com/en/latest/for_-_else.html#else-clause

Reply:
I'll also give a "nice" to Ambrus because I assume that's what he intended to do :)